packages/grid/data-grid/src/tests/rows.DataGrid.test.tsx

```diff
@@ -86,4 +86,17 @@ describe('<DataGrid /> - Rows', () => {
     fireEvent.click(document.querySelector('input[name="input"]'));
     expect(handleRowClick.callCount).to.equal(1);
   });
+
+  it('should apply the CSS class returned by getRowClassName', () => {
+    const getRowId = (row) => `${row.clientId}`;
+    const getRowClassName = (params) => (params.getValue('age') < 20 ? 'under-age' : '');
+    render(
+      <div style={{ width: 300, height: 300 }}>
+        <DataGrid getRowClassName={getRowClassName} getRowId={getRowId} {...baselineProps} />
+      </div>,
+    );
+    expect(getRow(0).className).to.contain('under-age');
+    expect(getRow(1).className).to.contain('under-age');
+    expect(getRow(2).className).not.to.contain('under-age');
```

Comment:
It will improve the failing error message

```suggestion
    expect(getRow(0)).to.have.class('under-age');
    expect(getRow(1)).to.have.class('under-age');
    expect(getRow(2)).not.to.have.class('under-age');
```